README.md

```diff
@@ -1,23 +1,71 @@
-# aerogear-dotnet-sdk
+# AeroGear Services Xamarin SDK
```

Comment:
not sure if we need the `Services` here, I think it's ok to just call it `AeroGear Xamarin SDK`

Reply:
ok, then, maybe we should also rename it on Android